### PR TITLE
Maintain same id for same phrase

### PR DESCRIPTION
**Pull request by roym899**
Small fix that introduces a global ID_FROM_PHRASE to maintain the same class id for the same phrase. Before the id would depend on the number of detected prompts, resulting in varying color when some phrase is / is not detected.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -22,6 +22,7 @@
 from segment_anything.modeling import Sam
 from groundingdino.models import GroundingDINO
 
+ID_FROM_PHRASE = {}
```
Comment (nikolausWest) [inline main.py]: Why did you need to make this a global variable? Too much work to pass around?

**Review by nikolausWest — APPROVED** [review]

File: main.py (hunk 1 above)
Comment (roym899) [reply main.py]: Felt like the easiest way, but I changed it since it's not too much work to avoid here.
